presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -384,6 +418,146 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
         return TableStatisticsMaker.getTableStatistics(typeManager, constraint, handle, icebergTable);
     }
 
+    public void createView(ConnectorSession session, ConnectorTableMetadata viewMetadata, String viewData, boolean replace)
+    {
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
```

Comment:
These are 99% the same as the HiveMetadata.createView. It would be better to extract these out as static methods in MetastoreUtil with an argument "metastore". Same for other view methods.
@tdcmeehan @agrawalreetika any other ideas?

Reply:
+1

Reply:
Will take care of this.